### PR TITLE
Adding support for console.log, console.warn, and greater customization

### PR DESCRIPTION
**Pull request by jantonso**
Getting it ready to be open-sourced.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,64 @@
...
 ```
-npm install @techchange/slack-webhooks
+npm install console-to-slack
 ```
 
 ## Usage
 Include this module:
 ```javascript
-const tcSlackWebhooks = require('@techchange/slack-webhooks')(settings);
+const consoleToSlack = require('console-to-slack')(options);
```
Comment (wchest) [inline README.md]: May want to either note that we're using ES6 syntax or switch to using `var`. I realize most people will know the difference, but might be good to be using more vanilla stuff in the docs.

Comment (jantonso) [inline README.md]: So would we want to write the whole thing in non ES6 then?

File: README.md [hunk 2]
```diff
@@ -1,33 +1,64 @@
...
 ```
 
-#### Settings
-For slack webhooks to work, settings must include:
+You can use the following methods:
 
-- **settings.slackWebhookUrl**: The url of the slack thread to send messages to.
-- **settings.name**: The name of the service. This is necessary to distinguish the location of the Slack message.
+- **consoleToSlack.overrideAll**: Overrides console.log, console.warn, and console.error to send messages to Slack via webhooks.
+- **consoleToSlack.overrideConsoleLog**: Overrides console.log to send error messages to Slack via webhooks.
```
Comment (wchest) [inline README.md]: What about naming these methods `overrideLog`, `overrideWarn`, `overrideError`? I don't think that it loses any semantic meaning and is shorter.

**Comment by wchest** [comment]
One last question, do we want to use public/private for method names, or just prefix private methods with `_`? Other than that, looks good @jantonso !

**Comment by jantonso** [comment]
I'm cool to switch to _

File: index.js [hunk 3]
```diff
@@ -1,47 +1,234 @@
...
+	}
+
+	/**
+	 * Overrides console.log to send the log messages to Slack via webhooks.
+	 * Note: Original console.log behaves as normal
+	 *
+	 * @function _overrideLog
+	 *
+	 * @since 0.0.2
+	 * @access public
```
Comment (matthewheck) [inline index.js]: Thanks to your effective use of the [Revealing Module Pattern](https://addyosmani.com/resources/essentialjsdesignpatterns/book/#revealingmodulepatternjavascript), these would be private now!

**Pull request merged**